packages/info/src/index.ts

```diff
@@ -13,70 +12,50 @@ interface Information {
     npmPackages?: string | string[];
 }
 
-interface ArgvI {
-    _?: string[];
-    bin?: boolean;
-    binaries?: boolean;
-    config?: string;
-}
-
 const DEFAULT_DETAILS: Information = {
     Binaries: ['Node', 'Yarn', 'npm'],
-    Browsers: ['Chrome', 'Firefox', 'Safari'],
+    Browsers: [
+        'Brave Browser',
```

Comment:
`Brave Browser`? What is it? :smile:

Reply:
https://brave.com/
A browser form a company of Brendan Eich ✨

Reply:
Do you know who use it? :smile:

Reply:
I use it (+edge chromium).
https://brave.com/brave-passes-10m-mau/

Reply:
Nevertheless, not a blocker for this PR anyways.